.github/workflows/ci.yml

```diff
@@ -0,0 +1,167 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+
+env:
+  PACKAGE_NAME: aws-iot-securetunneling-localproxy
+
+jobs:
+  osx:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: 'Checkout'
+    - name: Install brew dependencies
+      run: |
+        brew install openssl zlib cmake wget git
+    - name: Install boost
+      working-directory: ${{ github.workspace }}
+      run: |
+        wget https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.gz -O /tmp/boost.tar.gz
+        tar xzvf /tmp/boost.tar.gz
+        cd boost_1_69_0
+        ./bootstrap.sh --with-toolset=clang
+        ./b2 install toolset=clang
+    - name: Install protobuf
+      working-directory: ${{ github.workspace }}
+      run: |
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protobuf-all-3.6.1.tar.gz -O /tmp/protobuf-all-3.6.1.tar.gz
+        tar xzvf /tmp/protobuf-all-3.6.1.tar.gz
+        cd protobuf-3.6.1
+        mkdir build_make
+        cd build_make
+        cmake ../cmake
+        make
+        make install
+    - name: Install Catch2
+      working-directory: ${{ github.workspace }}
+      run: |
+        git clone https://github.com/catchorg/Catch2.git
+        cd Catch2
+        mkdir build
+        cd build
+        cmake ../
+        make
+        make install
+    - name: Building localproxy
+      working-directory: ${{ github.workspace }}
+      run: |
+        mkdir build
+        cd build
+        cmake .. -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1g/ -DOPENSSL_LIBRARIES=/usr/local/Cellar/openssl@1.1/1.1.1g/lib/
+        make
+  ubuntu:
+    runs-on: ubuntu-18.04
```

Comment:
If you do ubuntu-latest, I believe it's actually pointing to 18.04 rather than 20 (https://github.com/actions/virtual-environments)

Reply:
Updated the ubuntu version to 'ubuntu-latest'